roles/openshift_logging/tasks/oc_apply.yaml

```diff
@@ -18,6 +18,26 @@
   failed_when: "'error' in generation_apply.stderr"
   changed_when: no
```

Comment:
we could only get the name as output with `oc apply` so I moved the check to be immediately after the apply task we are evaluating